integration_tests/__tests__/__snapshots__/showConfig-test.js.snap

```diff
@@ -72,9 +72,6 @@ exports[`jest --showConfig outputs config info and exits 1`] = `
     "mapCoverage": false,
     "noStackTrace": false,
     "notify": false,
-    "projects": [
-      "/mocked/root/path/jest/integration_tests/verbose_reporter"
-    ],
```

Comment:
this was an intermediate `process.cwd()` value that was attached to a config in the process of parsing other configs. I change the code to pass it directly